plutus-scb/app/Main.hs

```diff
@@ -173,8 +173,8 @@ allServersParser =
         (pure
              (ForkCommands
                   [ MockNode
-                  , MockWallet
                   , ChainIndex
+                  , MockWallet
```

Comment:
does this change make a difference to anything? `Fork` would suggest to me that these are concurrent and so order should not matter?

Reply:
They are started up in the order shown, but we don't wait to see if the startup sequence completed.

So, it's an improvement in that the previous order was definitely wrong. Ideally we'd some mechanism where we start one and wait for a signal that it's running. But I honestly don't know how useful that will be when we're running in the real world, and not just against a bunch of simulators. So for now we just get this. :-D